modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java

```diff
@@ -1516,6 +1536,31 @@ private UserAccount _addAccountUserAccount(
 			accountId, userAccount);
 	}
 
+	private FileEntry _addImageFileEntry() throws Exception {
+		Company company = _companyLocalService.getCompany(
+			_userGroup.getCompanyId());
```

Comment:
probably you can use (also in the other test) the TestPropsValues (I don't if returns the company but for sure the companyId)

Reply:
@smottal yes, I tried but it did not work. But I can use the TestPropValues.getCompanyId() that works.